### ISSUE TITLE
Using-declaration with root namespace qualifier is not a hyperlink in source pages

### ISSUE THREAD

**Issue by eramongodb** [doxygen/doxygen#11230]
Given a header file `header.hpp` with the following content:

```cpp
namespace A {

struct X {};
struct Y {};

} // namespace A

namespace B {

using   A::X;
using ::A::Y; // <--

} // namespace B
```

In the source page for this header file (`header_8hpp_source.html`), only `A::X` in `using A::X;` is a link to a class page (in this case, `classA_1_1X.html`). The `::A::Y` in `using ::A::Y;` is _not_ a link despite a class page for both `A::Y` (`classA_1_1Y.html`) and `B::Y` (`classB_1_1Y.html`) being present.

Given the behavior of `using A::X;`, I expect the text for `using ::A::Y;` should be a link to `classA_1_1Y.html`. However, I think it is preferable for both to link to the class pages generated for namespace `B` rather than namespace `A`: `using A::X;` -> `classB_1_1X.html` and `using ::A::Y;` -> `classB_1_1Y.html` respectively.

Reproducible with Doxygen version 1.12.0 and with the latest commit (6a822eea5bf5a8deada330bdef3adf5de8a97675) on master. Observed during validation of fixes for https://github.com/doxygen/doxygen/issues/3760.

**Comment by doxygen** [comment]
@eramongodb Please verify if the referenced commit fixes the problem for you. Do not close the issue, this will be done automatically when the next official release becomes available.

**Comment by doxygen** [comment]
@albert-github not really on purpose, but I think it is ok, in the documentation also `A::Y` is used, not `::A::Y`

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
